### PR TITLE
Preliminary attempt to categorize labels into clusters

### PR DESCRIPTION
**Pull request by teolemon**
Preliminary attempt to categorize labels into clusters so that we can display them into wide categories in the website and the app.

### CODE REVIEW

**Comment by teolemon** [comment]
I would put `en:Environment` first. Perhaps `en:Origins` second. But to be honest, I was thinking geographic origins.

File: taxonomies/labels.txt [hunk 1]
```diff
@@ -175,6 +175,7 @@ hu:Szabad tartású
 nl:Vrije uitloop
 pt:Criação em liberdade
 ru:свободный выгул, открытый воздух
+label_categories:en: en:Animal welfare
```
Comment (hangy) [inline taxonomies/labels.txt:178]: `en:en:` seems redundant

File: taxonomies/labels.txt [hunk 2]
```diff
@@ -573,207 +579,6 @@ de:FSC Recycling
 fi:FSC-kierrätys
 fr:FSC Recyclé, FSC Recyclage
 
-<en:FSC
```
Comment (hangy) [inline taxonomies/labels.txt:576]: Why do you want to remove the FSC labels and how does that change relate to clustering?

File: taxonomies/labels.txt (hunk 1 above)
Comment (teolemon) [reply taxonomies/labels.txt:178]: it's the way it is for multilingual taxonomies embedded into other multilingual taxonomies.

File: taxonomies/labels.txt (hunk 2 above)
Comment (teolemon) [reply taxonomies/labels.txt:576]: unrelated. They were moved to packager codes, and this was a really tiny subset of all the FSC codes.

File: taxonomies/labels.txt (hunk 1 above)
Comment (stephanegigandet) [reply taxonomies/labels.txt:178]: That's the way it currently works when a property value is an entry in another taxonomy.
e.g. for additives, we have:
additives_classes:en: en:colour

That means that the English value for the additives_classes property is the entry en:colour.

The properties in taxonomy must have a language (and the value can be different based on that language). 

We should do this though:

label_categories:en: en:animal-welfare

Comment (teolemon) [reply taxonomies/labels.txt:178]: I'm confused @stephanegigandet in other occurrences I could swear you've asked me to put the real value and not the tag. Should I create a taxonomy of label clusters ?

File: taxonomies/labels.txt (hunk 2 above)
Comment (teolemon) [reply taxonomies/labels.txt:576]: @hangy should I remove them in a separate PR or are you ok with getting rid of them in this one ?

**Pull request merged**